libcloud/compute/drivers/vcloud.py

```diff
@@ -2186,3 +2186,34 @@ def _perform_snapshot_operation(self, node, operation, xml_data, headers):
         self._wait_for_task_completion(res.object.get('href'))
         res = self.connection.request(get_url_path(node.id))
         return self._to_node(res.object)
+
+    def ex_acquire_mks_ticket(self, vapp_or_vm_id, vm_num=0):
+        """
+        Retrieve a mks ticket that you can use to gain access to the console
+        of a running VM.
+        :param vapp_or_vm_id: vApp or VM ID you want to connect to.
```

Comment:
include :type as well.

separate params with spaces so that the docs format correct
specify rtype.